pyquery/pyquery.py

```diff
@@ -169,7 +169,10 @@ def __init__(self, *args, **kwargs):
         if kwargs:
             # specific case to get the dom
             if 'filename' in kwargs:
-                html = open(kwargs['filename'])
+                if 'encoding' in kwargs:
+                    html = open(kwargs['filename'], encoding=kwargs['encoding'])
```

Comment:
Why not just `html = open(kwargs['filename'], encoding=kwargs.get('encoding'))` instead of the if / else?

Reply:
>Why not just html = open(kwargs['filename'], encoding=kwargs.get('encoding')) instead of the if / else?

Because I imitated `if 'filename' in kwargs`. But your suggestion is simpler than my code, so I adopt your suggestion.